server/src/main/resources/available.toggles

```diff
@@ -50,6 +50,11 @@
       "key": "show_old_comparison_spa",
       "description": "Switch to the old comparison SPA. Default is false.",
       "value": false
+    },
+    {
+      "key": "new_pipeline_config_spa",
+      "description": "Switch to the new pipeline edit SPA. Default is false.",
```

Comment:
How about: `When enabled, provides an option on the pipeline config rails page to use the quick edit SPA.` 
Cause currently, it does not switch, just shows a button to go to the new SPA!

Reply:
Done